help/dependencies.txt

```diff
@@ -134,7 +134,7 @@ Local Maven repository Lucene artifacts (for use with the
 "-Plucene.dev.version" project property) may be created by running the
 following command from within a local Lucene project:
 
-  gradlew mavenToLocalRepo
+  gradlew mavenToLocal
```

Comment:
ref: https://github.com/apache/lucene/blob/releases/lucene/9.9.1/gradle/maven/maven-to-local-m2.gradle#L22 or https://github.com/search?q=repo%3Aapache%2Flucene+mavenToLocal&type=code